src/phase.c

```diff
@@ -241,6 +241,13 @@ void mag_to_phase(float phase[],             /* Nfft/2+1 output phase samples in
     int  Ns = Nfft/2+1;
     int  i;
 
+    /* initialize array */
+
+    for(i=0; i<Nfft; i++) {
+        Sdb[i].real = 0.0;
+        Sdb[i].imag = 0.0;
+    }
+
```

Comment:
Hi @srsampson - so why is this needed given the next few lines that fill ```Sdb[]```?  Do they miss some values or did the compiler complain?

Reply:
I believe the next few lines only initialize 0 - Ns while the array is 0 - Nfft thus half the values I assume will be random stack values.

This seemed suspicious to me.

Reply:
> I believe the next few lines only initialize 0 - Ns while the array is 0 - Nfft thus half the values I assume will be random stack values.
> 
> This seemed suspicious to me.

Oh darn, I have made a BIG mistake here. I completely missed the code inside the loop!

I feel maybe this whole PR is a wash. Probably needs deleted. argh!